CHANGES.txt

```diff
@@ -1,4 +1,59 @@
-﻿NUnit 3.0.0 Beta 2 - May 12, 2015
+﻿NUnit 3.0.0 Beta 3 - July ??, 2015
+
+Framework
+
+ * The RangeAttribute has been extended to support more data types including 
+   uint, long and ulong
+ * Added platform support for Windows 10 and fixed issues with Windows 8 and
+   8.1 support
+ * Added async support to the portable version of NUnit Framework
+ * Several TestCaseSourceAttribute constructors have been deprecated, see the
+   documentation for more information
```

Comment:
We decided not to do this and instead require those constructors to be static. I forgot to change the title of the issue, but just did. Also applies to ValueSourceAttribute.